rotkehlchen/chain/ethereum/modules/uniswap/v3/decoder.py

```diff
@@ -557,5 +558,8 @@ def enricher_rules(self) -> list[Callable]:
     def counterparties(self) -> list[str]:
         return [CPT_UNISWAP_V3]
 
-    def post_decoding_rules(self) -> list[tuple[int, Callable]]:
-        return [(0, self._routers_post_decoding)]
+    def post_decoding_rules(self) -> PostDecodingrules:
+        return PostDecodingrules(
+            common=[(0, self._routers_post_decoding)],
```

Comment:
No as I said the "common" makes no sense. Post decoding rules should only exist if the decoder runs.